core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -49,13 +54,32 @@ func IsRollupWithL1Support(chainType chaintype.ChainType) bool {
 	return slices.Contains(supportedChainTypes, chainType)
 }
 
-func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle) (L1Oracle, error) {
+func IsDAClientSupported(clientsByChainID map[string]DAClient, DAChainID string) bool {
```

Comment:
nit: where is this method being used? Don't think I see a reference anywhere in this PR